docs/CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `ETH_MIN_GAS_PRICE_WEI` configuration option. This defaults to 1Gwei on mainnet. Chainlink will never send a transaction at a price lower than this value.
 
+- Add `chainlink node db migrate` for running database migrations. It's
+  recommended to use this and set `MIGRATE_DATABASE=false` on more complex
```

Comment:
"on more complex deployments" is a little vague - let's just say "if you want to run the migrations separately outside of  application startup"